tests/phpunit/integration/Core/Assets/AssetsTest.php

```diff
@@ -221,23 +229,25 @@ function() {
 	}
 
 	public function test_base_data__product_base_paths__no_permalink_structure() {
+		$this->enable_feature( 'userInput' );
 		$data = $this->get_inline_base_data();
 		$this->assertTrue( empty( $data['productBasePaths'] ) );
 	}
 
 	public function test_base_data__product_base_paths__empty() {
-		update_option( 'permalink_structure', '/%postname%/' );
+		$this->enable_feature( 'userInput' );
+		$this->set_permalink_structure( '/%postname%/' );
```

Comment:
This helper method from the base WP testcase is important to use because it interacts with WP_Rewrite under the hood, without which will result in unexpected results, particularly regarding "extra" permalink structs that are added by post types during registration.